hr_expense_analytic_require/models/company.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    default_expense_analytic_id = fields.Many2one(
+        'account.analytic.account',
```

Comment:
```suggestion
        'account.analytic.account',
        required=True,
```

Reply:
This is being added in the view layer.
In there a significant benefit in having it enforced at the model layer?
In this case, we could get away with setting it here, because of the default, but in general doing so can break other modules.